Project.toml

```diff
@@ -1,27 +1,27 @@
 name = "TextParse"
 uuid = "e0df1984-e451-5cb5-8b61-797a481e67e3"
-version = "1.0.2-DEV"
+version = "1.0.2"
```

Comment:
In general these should not be updated in a "normal" commit, otherwise it throws off the package tagging functionality in VS Code. So, I just merged https://github.com/queryverse/TextParse.jl/pull/164 instead :)